### ISSUE TITLE
1.0.42 Crashing on iPhone 6 after merchant share

### ISSUE THREAD

**Issue by tomtheall** [ventuso/Fi-Test#100]
iOS 8.1.2
iPhone 6
Fi version 1.0.42

Shared my first merchant network (The Hill) and now app is crashing immediately every time I open the app. It goes to the map page (discover) and crashes.

Not sure how else to diagnose/troubleshoot — sometimes I can bypass the crash by quickly hitting "build" button...if that helps. But then when I go back to discover it crashes.

Can't use. Tried logging out, restarting the app, restarting the phone, reinstalling the app, etc.

**Comment by tomtheall** [comment]
Follow up: I downloaded 1.0.43 ...slightly more stable but still crashing when:
1) When I tap on the pop up for the merchant I shared (The Hill)
2) When I try viewing The Hill network in the list view.
3) When I am connected (via WiFi) to The Hill

1 & 2 makes me think that since I haven't uploaded a picture yet for the background, the app doesn't know what to display for the list view so it just quits/terminates.

Not sure why number 3 triggers the crash. (I put in a fake password if that has anything to do with it.)
